cts/cts-cli.in

```diff
@@ -2289,6 +2289,66 @@ function test_tools() {
     desc="Switch to nonexistent shadow instance (force) (XML)"
     cmd="crm_shadow --switch $shadow --batch --force --output-as=xml"
     test_assert_validate $CRM_EX_NOSUCH 0
+    
+    CIB_file_invalid_1="$test_home/cli/crm_verify_invalid_bz.xml"
+    CIB_file_invalid_2="$test_home/cli/crm_verify_invalid_no_stonith.xml"
+    CIB_file_invalid_3="$test_home/cli/crm_verify_invalid_xml_error.xml"
+    
+    desc="Verify a file-specified invalid configuration, outputting as xml"
+    cmd="crm_verify --xml-file '$CIB_file_invalid_1' --output-as=xml"
+    test_assert_validate $CRM_EX_CONFIG 0
+
+    desc="Verify another file-specified invalid configuration, outputting as xml"
+    cmd="crm_verify --xml-file '$CIB_file_invalid_2' --output-as=xml"
+    test_assert_validate $CRM_EX_CONFIG 0
+
+    desc="Verify a file-specified invalid configuration with an XML error, outputting as xml"
+    cmd="crm_verify --xml-file '$CIB_file_invalid_3' --output-as=xml \
+      |& sed 's#\(/.*/\)crm_verify_invalid_xml_error.xml#crm_verify_invalid_xml_error.xml#' \
+      |& cut -b-253"
```

Comment:
This triggers another issue ... if you look at the giant sed after the comment starting with "last-rc-change", it removes everything in `<cib>` after `cib-last-written`, but it checks for the closing `>` which no longer exists :-/ So, now that pattern is not matching when it should.

Also, if we truncate here, it might break the XML output so that it doesn't validate against the schema. I think we need to do the truncation just before diff'ing against the reference output at the end.

BTW I checked the existing reference text, and there are many lines >253 characters, so I'm perplexed why this hasn't been a problem before.